From 392f705b702a48f8488cb0648041e9987282e622 Mon Sep 17 00:00:00 2001
From: Sameer Mulla <sameer.mulla@lge.com>
Date: Tue, 10 Dec 2019 16:15:43 +0530
Subject: [PATCH] Fix volume level notification not appearing after 127

:Release Notes:
Fix volume level notification not appearing after 127

:Detailed Notes:
Even volume level should be notified even its same
value as previous incase it reached lower value 0
or upper value 127, so that UI can be updated that
max or min volume is reached

:Testing Performed:
Builded and Tested

:QA Notes:

:Issues Addressed:
[PLAT-97324] There is an environment where avrcp / getRemoteVolume
             does not return.
---
Upstream-Status: Inappropriate [webos specific]

 profiles/audio/media.c     | 4 ++--
 profiles/audio/transport.c | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/profiles/audio/media.c b/profiles/audio/media.c
index c434043e1..c39b1846a 100644
--- a/profiles/audio/media.c
+++ b/profiles/audio/media.c
@@ -2039,8 +2039,8 @@ static void media_player_set_volume(int8_t volume, struct btd_device *dev,
 {
 	struct media_player *mp = user_data;
 
-	if (mp->volume == volume)
-		return;
+	/*if (mp->volume == volume)
+		return;*/
 
 	mp->volume = volume;
 }
diff --git a/profiles/audio/transport.c b/profiles/audio/transport.c
index bcfee9140..64e0b8dcc 100644
--- a/profiles/audio/transport.c
+++ b/profiles/audio/transport.c
@@ -2085,8 +2085,8 @@ void media_transport_update_volume(struct media_transport *transport,
 		return;
 
 	/* Check if volume really changed */
-	if (a2dp->volume == volume)
-		return;
+	/*if (a2dp->volume == volume)
+		return;*/
 
 	a2dp->volume = volume;
 
